src/libraries/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.Date.cs

```diff
@@ -149,7 +149,7 @@ public static void TestingStringsInvalidConversionToDateTimeOffset(string jsonSt
         }
 
         [Fact]
-        // https://github.com/dotnet/corefx/issues/39067.
+        // https://github.com/dotnet/runtime/issues/30095.
         public static void Regression39067_TestingDateTimeMinValue()
```

Comment:
Same issue here (test name with issue number no longer matches, we should remove the number).